tests/system/model_monitoring/test_model_monitoring.py

```diff
@@ -287,13 +287,17 @@ def test_basic_model_monitoring(self):
             )
             sleep(choice([0.01, 0.04]))
 
+        # Wait for model endpoint stats to be updated
```

Comment:
I suggest to actively check that the stats are updated with a retry mechanism instead of waiting a magic time.
Check out tests.system.runtimes.test_notifications.TestNotifications.test_run_notifications for reference